### ISSUE TITLE
Contact Page - When we change tag from contact view action button it does not reflect immediately

### ISSUE THREAD

**Issue by Avinash-ecb** [Fammeo/EasyCloudBooks#1749]
**Describe the bug**
- When we change tags from contact view action button it does not change tags immediately we have to close contact view and again open to see changes 

![image](https://user-images.githubusercontent.com/46588125/196344030-6af650a3-733b-42cc-918b-d9699644a6bf.png)

https://www.berrycast.com/conversations/4900df13-53b8-5563-a033-79ac2de8dbfd

**Comment by mitulpanch** [comment]
I think we need too much of coding to do this
Consider this as action module remake,
this was not planned / not developed specifically to do certain task

Not impossible to do, but requires lot time than it gives perfection to user

**Comment by mitulpanch** [comment]
forwarding to other for comments

**Comment by Avinash-ecb** [comment]
Sir I think if we not give edit, remove and change tag option in this action button it is okay, because in contact view it is already given If he want to do he can do directly from there he will not go action button
This is my point

**Comment by mitulpanch** [comment]
ok @Avinash-ecb , i think you mint to say that change tag , other tag related things not required on Action button

**Comment by Avinash-ecb** [comment]
Yes @mitulpanch sir, and also edit contact, and remove contact button also because it is already in contact view
![image](https://user-images.githubusercontent.com/46588125/196368772-05bc1572-efa6-4efb-92a6-d9dd77e21253.png)

**Comment by mitulpanch** [comment]
This is not coming, this is temporary, as currently only released on 74, so outside buttons are not removed,
once testing is done, then we will remove outside buttons

**Comment by vinpahilwani** [comment]
mujhe isme kya decide karna hai?
